src/WebJobs.Script.WebHost/Diagnostics/SystemLogger.cs

```diff
@@ -17,23 +17,31 @@ public class SystemLogger : ILogger
         private readonly bool _isUserFunction;
         private readonly string _hostInstanceId;
         private readonly IEnvironment _environment;
+        private readonly LogLevel _logLevel;
+        private readonly IDebugStateProvider _debugStateProvider;
 
-        public SystemLogger(string hostInstanceId, string categoryName, IEventGenerator eventGenerator, IEnvironment environment)
+        public SystemLogger(string hostInstanceId, string categoryName, LogLevel logLevel, IEventGenerator eventGenerator, IEnvironment environment, IDebugStateProvider debugStateProvider)
```

Comment:
It looks like we'll never change the default `LogLevel` here (except in a test) -- can we remove this parameter and have the default set inside the class?

Reply:
Fixed